Sources/FoundationExtensions/Optional+Extensions.swift

```diff
@@ -44,14 +44,3 @@ extension OptionalType {
     }
 
 }
-
-// MARK: -
-
-internal extension Optional where Wrapped == String {
```

Comment:
Moved this to `String+Extensions` since it makes more sense there. This file is also now sym-linked so it can be used by `RevenueCatUI`.